### ISSUE TITLE
the installation of venv process is not deployed on prod

### ISSUE THREAD

**Issue by 12rambau** [openforis/sepal#277]
in it's current implementation, the installation of venv is broken as this PR is not yet included:  https://github.com/openforis/sepal/pull/269

@yfinegold, @BuddyVolly, @lecrabe, @dfguerrerom: As the SEPAL course is ongoing, we should not make new releases just now so let me know if specific applications are broken so that we perform specific workarounds.

**Comment by 12rambau** [comment]
ipywidgets need to be forced as well (if not, module will not be found)

I need to check the earthengine-api issue, it's still installing 0.1.350

**Comment by dfguerrerom** [comment]
what do you mean with forced? forced in requirements?

**Comment by 12rambau** [comment]
yep, pin the version of ipywidgets to be the same as the one in the jupyter server.
There will be no issue with voila as it's 100% relying on the venv to render the widgets
